webauthn-rs/src/lib.rs

```diff
@@ -1161,6 +1169,17 @@ impl Webauthn {
             .user_verification_policy(UserVerificationPolicy::Required)
             .reject_synchronised_authenticators(true)
             .exclude_credentials(exclude_credentials)
+            .hints(Some(
+                // hybrid can NOT perform attestation
```

Comment:
**nit:** `s/can/do/` (also below)

I believe they technically *could*, I can share a USB FIDO2 device over caBLE. Just Apple and Google's implementations (with synchronised credentials stored in iCloud Keychain / Google Password Manager) don't.

Also, the documentation of this function says that it "precludes the use of (synchronised) authenticators"; this should be reworded to be more specific. Something like:

----

Hardware attestation requires that private key material is bound to a single hardware authenticator, and cannot be copied or moved out of it. At present, *all* widely deployed `Hybrid` authenticators (Apple iCloud Keychain and Google Passkeys in Google Password Manager) are synchronised authenticators which can roam between multiple devices, and so *can never be attested*.

**As of `webauthn-rs` vX.Y**, this creates a challenge with "hints" `(link to PublicKeyCredentialHints)` to only use `ClientDevice` or `SecurityKey` devices, so a user-agent supporting Webauthn L3 won't offer to use `Hybrid` credentials. On user-agents not supporting Webauthn L3, and on older versions of `webauthn-rs`, user-agents would show a QR code and a user could attempt to register a `Hybrid` authenticator, but it would always fail at the end -- which is a frustrating user experience!